README.md

```diff
@@ -217,6 +217,10 @@ kubectl create -f scope-probe-ds.yaml
 
 ## <a name="developing"></a>Developing
 
+Building scope from source depends on the latest versions of
+[docker](https://www.docker.com/) and [go](https://golang.org/), so
```

Comment:
Not entirely. `make deps` does `go get github.com/mjibson/esc` which is used to build `prog/static.go` (which is not containerized).